plugins/labhub.py

```diff
@@ -271,6 +271,16 @@ def register_check(function):
             return function
 
         if self.GH_ORG_NAME == 'coala' and self.GL_ORG_NAME == 'coala':
+            @register_check
+            def block_gci_issue_assignment(user, iss):
+                """
+                False if the issue is labelled as 'initiatives/gci'.
+                """
+                if 'initiatives/gci' in iss.labels:
+                    return False
+                else:
+                    return True
+            
```

Comment:
The code does not comply to PEP8.

*Origin: PEP8Bear, Section: `all.autopep8`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmp_tbotk3e/plugins/labhub.py
+++ b/tmp/tmp_tbotk3e/plugins/labhub.py
@@ -280,7 +280,7 @@
                     return False
                 else:
                     return True
-            
+
             @register_check
             def difficulty_level(user, iss):
                 """
```